Verifying-ERC20.md

```diff
@@ -133,7 +131,7 @@ which means there is one sequence of transactions (i.e., create, transfer) which
 
 ## Checking overflows and underflows
 
-As a bonus track, lets suppose a much simpler specification stating that after transferring tokens the `sender` [must has less or equal amount of tokens](https://github.com/garbervetsky/ERC-Verisol-Demo/blob/master/ERC20-uf.sol#L76).
+As a bonus track, let us suppose a much simpler specification stating that after transferring tokens the `sender` [must has less or equal amount of tokens](https://github.com/garbervetsky/ERC-Verisol-Demo/blob/master/ERC20-uf.sol#L76).
```

Comment:
If you introduce this properties already in one place, then you don't have to add the new property, only talk about the flag.